WhaleSpotting/Controllers/SpeciesController.cs

```diff
@@ -0,0 +1,26 @@
+using Microsoft.AspNetCore.Mvc;
+using WhaleSpotting.Models.Response;
+using WhaleSpotting.Services;
+using WhaleSpotting.Models.Request;
+
+namespace WhaleSpotting.Controllers;
+
+[ApiController]
+[Route("species")]
+public class SpeciesController : ControllerBase
+{
+    private readonly ISpeciesService _speciesService;
+    
+    public SpeciesController(ISpeciesService speciesService)
+    {
+        _speciesService = speciesService;
+    }
+
+    [HttpGet("")]
+    public ActionResult<SearchResponse> Search([FromQuery] SpeciesSearchRequest speciesRequest)
```

Comment:
We don't need this `SpeciesResponse` model, we can just return the `List<SpeciesResponse>` here